src/backend/src/routes/teams.routes.ts

```diff
@@ -39,4 +39,6 @@ teamsRouter.post(
   TeamsController.createTeam
 );
 
+teamsRouter.delete('/:id/archive');
```

Comment:
should be `:teamId`

Reply:
I know the ticket said to make this a delete endpoint, but since we're even calling the endpoint archive instead of delete, I think it makes more sense to make it a post endpoint